composer.json

```diff
@@ -29,6 +29,7 @@
 		"nette/caching": "^3.0",
 		"nette/di": "^3.0.0",
 		"orisai/coding-standard": "^2.0.0",
+		"phpbench/phpbench": "^1.0",
```

Comment:
`^1.0.0`
- it's not that case, but Composer has some odd cases with `x.y` syntax